lib/cJSON.c

```diff
@@ -91,24 +92,24 @@ typedef struct {
 } error;
 static error global_error = { NULL, 0 };
 
-CJSON_PUBLIC(const char *) cJSON_GetErrorPtr(void)
```

Comment:
What about we keep the original function names? While I understand this could be an already implemented change working in previous cJSON.

Reply:
Yes, this is an already implemented change working in previous cJSON. The reason for that change was to avoid linking conflict: https://github.com/snowflakedb/libsnowflakeclient/pull/73
I think the reason might be that we have another copy of cJSON in curl, without changing the original function names, we would have duplicated implementation for the same functions when linking to both curl and libsnowflakeclient libs.